desktop-exporter/otlp_payload_processor_test.go

```diff
@@ -45,13 +45,13 @@ func TestExtractSpans(t *testing.T) {
 	assert.Equal(t, expectedSpanID, spans[0].SpanID)
 	assert.Equal(t, expectedParentSpanID, spans[0].ParentSpanID)
 	assert.Equal(t, "span", spans[0].Name)
-	assert.Equal(t, "SPAN_KIND_INTERNAL", spans[0].Kind)
+	assert.Equal(t, "Internal", spans[0].Kind)
```

Comment:
The way these are turned into strings has changed: https://github.com/open-telemetry/opentelemetry-collector/blob/main/pdata/ptrace/traces.go#L99-L137

Reply:
Oh cool I should remove this from the utils on the front-end!